src/js/components/ContentCard.js

```diff
@@ -0,0 +1,170 @@
+// (C) Copyright 2014-2015 Hewlett Packard Enterprise Development LP
+
+import React, { Component, PropTypes } from 'react';
+import classnames from 'classnames';
+import Props from '../utils/Props';
+import Box from './Box';
+import Tile from './Tile';
+import Heading from './Heading';
+import Paragraph from './Paragraph';
+import Anchor from './Anchor';
+import Layer from './Layer';
+import Video from './Video';
+import WatchIcon from './icons/base/Watch';
+
+const CLASS_ROOT = 'content-card';
+
+export default class ContentCard extends Component {
+  constructor (props) {
+    super(props);
+    this._handleClick = this._handleClick.bind(this);
+    this.state = {
+      activeVideo: false
+    };
+  }
+
+  _handleClick (event) {
```

Comment:
Everywhere else in grommet, we use the term "on", rather than "handle" -> `_onClick`